bigflow/log.py

```diff
@@ -93,12 +99,54 @@ class LogConfigDict(TypedDict):
     log_level: typing.Union[str, int]
 
 
-def _generate_cl_log_view_link(params: dict):
-    query = quote_plus("\n".join(
-        '{}="{}"'.format(k, v)
+def _generate_cl_log_view_query(params: dict, separator: str = '='):
+    query = "".join(
```

Comment:
nit: it is better to explicitly specify "join separator", in our case `"\n".join(...)`

Reply:
removed separator arg